test/renate/data/test_data_module.py

```diff
@@ -134,6 +134,7 @@ def test_tiny_imagenet_data_module(tmpdir):
     assert train_data[0][0].size() == test_data[0][0].size() == (3, 64, 64)
 
 
+@pytest.mark.slow
```

Comment:
why is this slow? is it the data download? can we use a smaller dataset? https://huggingface.co/datasets/ethos

Reply:
Rotten tomatoes download is slow.